server side/router/websites.router.js

```diff
@@ -0,0 +1,10 @@
+import  express from 'express';
+import { getWebsitesById,getAllWebsites,addWebSites,UpdateWebSites,deleteWebsites} from '../controllers/websites.controller.js';
+ const router=express.Router();
+router.get('/getAllWebsites',getAllWebsites);
+router.get('/getWebsitesById/:id',getWebsitesById);
+router.put('/UpdateWebSites/:id',UpdateWebSites);
+router.post('/addWebSites',addWebSites);
+router.delete('/deleteWebsites/:id',deleteWebsites);
```

Comment:
remove the create-update-delete words, look in the user router for the correct way to do it.